src/components/Shared/CrossingMapPage/CrossingMapSidebar.js

```diff
@@ -165,7 +165,9 @@ class CrossingMapSidebar extends Component {
                 setSelectedLocationCoordinates={setSelectedLocationCoordinates}
                 toggleSearchFocus={this.toggleSearchFocus}
                 communities={allCommunities}
-                communityId={currentUser && currentUser.communityId}
+                communityId={(currentUser && 
+                              currentUser.role !== 'floods_super_admin') ?
```

Comment:
these roles feel like they should be constants living somewhere. seems brittle if the string we used to describe a super_admin ever changed.